src/styles/settings.scss

```diff
@@ -3,6 +3,10 @@ $white: #fff;
 $elephant: #0F2B3B;
 $robinsEggBlue: #00BDB5;
 $osloGray: #838C93;
+$alto: #DBDBDB;
+$firefly: #09141A;
+$vida-loca: #4B9217;
```

Comment:
Are we going to use camel case or snake case?

Reply:
aah sorry. `camelCase` I think

Reply:
Not really sure if I like this naming 🤔 
Maybe something to think about. From the name I can not say the colour it is. Maybe something more project related (like `primaryColor` or `titleGray`) or even colour related (`deepBlue`, `paleBlue`) could be easier to remember.
Nothing to do with this PR though, just opening the debate.

Reply:
The thing is if you've got couple of shades of the same color you quickly run out of names for them (i.e light-blue, dark-blue, medium-blue, ...).
What I was doing in other projects if some of the colors were used more frequently than the others, then I'd give them a more semantic name like `theme-color` or `primary-blue` or sth similar. i.e.:
```
// all colors
$robinsEggBlue: #00BDB5;
$alto: ...
...

// frequently used
$widgets-color: $robinsEggBlue;
...
```
Would that help?

Reply:
@aabdaab That's a nice approach, in the past, I also had problems with colors naming e.g. (grey, darkGrey, darkestGrey, darkAsHellGrey), the problem is that at some point this naming convention can be misleading either. But your solution above makes sense to me :+1: